docs/getting-started.md

```diff
@@ -9,47 +9,27 @@ route: /getting-started
 
 _5 mins read_
 
-This doc will help you to get up and running with the library in your project.
+This doc will help you to get up and running with the library in an existing project.
 
-The main component of this library is the `@uireact/foundation`, this package has the definition of types, enums, objects and 
+The main component of this library is the `@uireact/foundation`, this package has the definition of types, enums, objects and
 helper functions that other packages across the library need, it also has all the handling that is needed for the themes, responsive design and dialogs.
 
 
-
-## 1. Install peer dependencies 🛠️
```

Comment:
peer deeps are automatically installed starting on npm v7.0.0 https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/